decanter_ai_sdk/non_blocking_client.py

```diff
@@ -298,21 +308,21 @@ def train_ts(
         datetime: str,
         time_groups: List,
         timeunit: TimeUnit,
```

Comment:
[TODO] Remove the duplicate default values for all the parameters set in `train_ts` under the client.py.

Reply:
Thanks for the suggestion!
I've created a new issue to record this to-be-solved issue (https://github.com/MoBagel/decanter-ai-sdk/issues/64)